### ISSUE TITLE
Time of last completed update

### ISSUE THREAD

**Issue by Morgensohn** [drpout/boilr#258]
Ir woul be useful, at least for users with irregular connectivity, to know to what time do the exchange rates refer, an indication of last time of completed update..

**Comment by dllud** [comment]
Agreed. We do keep that info on the alarm. It's now a matter of deciding where we should present it. Do you have any suggestion?

On the main activity (screen) there is no space left. The only place I can think of is the Alarm Settings, bellow Update interval. Though, this is not a setting... perhaps we should think about creating an Alarm Info view/section.

**Comment by Morgensohn** [comment]
Thanks for the reply. I agree there is little or no space left on the main activity screen, though I tried to imagine where I could find the info there (see .png).

Another solution could be using colours: having a setting saying if an update interval is skipped for x minutes, hours, etc., the currency pair, i.e. the info on the left side, would go orange, or another alarming colour (or have an exclamation sign, something to indicate that it's gone stale).

![screenshot_2017-07-05-09-00-41](https://user-images.githubusercontent.com/26673136/27862468-8d1f4b42-6185-11e7-89ba-142805bb7feb.png)

**Comment by dllud** [comment]
Thanks for the input.

On principle I am against using only colours to convey information:
- it forces the user to learn a new code (what each colour means);
- daltonic people cannot see it.
On this specific case it would also create the "angry fruit salad" visual effect as we already use green and red for the price trend.

I would rather opt for your first solution, perhaps replacing the word "updating" with a refresh icon.

Though, I still think this may be an unnecessary interface clutter. We already have a "no internet warning" that shows up on the notification bar whenever an alarm can't update due to missing internet connection. You can enable it on the app settings. Perhaps we can just add the last update time over there.

**Comment by Morgensohn** [comment]
Thanks, I agree with your perspective on the colours. 

Leaving the time info on the warning could be a solution (but one of the first settings I changed on Boilr was to switch off the warning, as it was, for some reason, coming up too often). But the solution of showing some time info on the warning would only come up if there is a warning triggered, and would show one time info for everything, no? 

Perhaps having the info more fine grained on each currency pair would be more useful, accurate etc. 

One reason for this might be that info can become stale not because of the lack of internet connection, but because of problems with the exchanges from which the info is taken (am I correct it is taken individually from each exchange using their own API? or is it polled from a central website which in turn gets the data from each exchange?)

**Comment by dllud** [comment]
> am I correct it is taken individually from each exchange using their own API?

Yes. Boilr contacts all exchanges directly. No data is exchanged with third-party servers.


> having the info more fine grained on each currency pair would be more useful

Yes, considering your reasons above, I have to agree.

What still bugs me is cluttering even more the alarm list. Let's wait and see what ideas others may have about this... @algazarra @andrefbsantos ?
